diode/client.go

```diff
@@ -70,20 +75,31 @@ func parseTarget(target string) (string, string, bool, error) {
 	}
 
 	if !allowedSchemesRe.MatchString(u.Scheme) {
-		return "", "", false, errors.New("target should start with grpc:// or grpcs://")
+		return "", "", false, ErrInvalidTargetScheme
 	}
 
 	authority := u.Host
 	if u.Port() == "" {
-		authority += ":443"
+		switch u.Scheme {
+		case "grpc", "http":
+			authority += ":80"
+		case "grpcs", "https":
+			authority += ":443"
+		default:
+			return "", "", false, fmt.Errorf("missing port with unsupported scheme: %s: %w", u.Scheme, ErrInvalidTargetScheme)
+		}
 	}
```

Comment:
This changes the default behavior for `grpc://` to have port `80` used instead of `443`.